src/skai/beam_utils.py

```diff
@@ -148,7 +151,9 @@ def get_pipeline_options(
       'sdk_location': 'container',
       'setup_file': _get_setup_file_path(),
       'max_num_workers': max_workers,
-      'use_public_ips': False  # Avoids hitting public ip quota bottleneck.
+      'use_public_ips': False,  # Avoids hitting public ip quota bottleneck.
+      'worker_machine_type': worker_machine_type,
```

Comment:
@panford I don't see this option listed here:

https://cloud.google.com/dataflow/docs/reference/pipeline-options#python

Should it be "machine_type" instead? (machine_type is already one of the options we support on line 160.

Reply:
Hi @jzxu Thanks for pointing this out. I was using a slightly older naming convention. Can we unify the two options? Do we use worker_type or machine_type (this works on our end)?

Reply:
We are already using machine_type. So let's just stick with that.